### DISCUSSION TITLE
Create a row using public api with attachment

### DISCUSSION THREAD

**Discussion by JamshedAlamQaderi** [Budibase/budibase#9662] [Help] [answered]
Hi Budibase Team,

You guys developed a great application which is easy to build web applications. I'm making a web app. From android app i want to create a row using attachments. Could anyone tell me how i can do that?

**Comment by melohagan** [comment, marked as answer]
Hey @JamshedAlamQaderi 

So you would like to create a new row in an Internal Budibase DB with an attachment - is that right?

Have you run into any issues or are you just unsure?

Here's some docs pages that may be useful:
 - https://docs.budibase.com/docs/attachments
 - https://docs.budibase.com/docs/attachment

**Reply by JamshedAlamQaderi** [reply]
I want to upload a complete row with attachment using API not in the app. I've tried with postman and made request using formdata and row created but I'm not able to see the attachment

**Reply by melohagan** [reply]
Ah sorry, I missed the title of the discussion!

I'm not sure if it is currently possible to send an attachment with the public API. 

I found these discussions which may have some relevant info:
 - https://github.com/Budibase/budibase/discussions/9168
 - https://github.com/Budibase/budibase/discussions/9190
 - https://github.com/Budibase/budibase/discussions/3725

**Reply by JamshedAlamQaderi** [reply]
Thank you for the references. Last one helped me

**Reply by Aculeo** [reply]
Can we get an API that works also in the cloud version? Uploading attachments through API seems crucial to me. For example when invoices are generated in a different system, and then need to be written back to Budibase.
